internal/pkg/selinux/selinux.go

```diff
@@ -34,7 +34,13 @@ var IsEnabled = sync.OnceValue(func() bool {
 
 	val := procfs.ProcCmdline().Get(constants.KernelParamSELinux).First()
 
-	return val != nil && *val == "1"
+	var selinuxFSPresent bool
+
+	if _, err := os.Stat("/sys/fs/selinux"); err == nil {
```

Comment:
just checking, does it depend only on `sysfs` being mounted?

Reply:
yeh, I checked an upgrade path from 1.9 with `security=none` set